README.md

```diff
@@ -6,6 +6,25 @@ ALF(Alice Low-level Frontend) spawns DIM services as an interface with detector
 ## Requirements
 In order to run ALF a DIM Nameserver has to be up and running. For performance reasons, it is recommended to run the DIM Nameserver on the host running the FRED server.
 
+## Install ALF
```

Comment:
This should not end up in the ALF README.

Reply:
I reverted them back, I was unaware that It was done on the master.

Reply:
No worries, thanks! I'm merging.

Reply:
The README changes were meant to be for AliceMCH repo but it is not that important, I can have them saved locally.